.github/workflows/pr.yml

```diff
@@ -24,8 +24,7 @@ jobs:
         run: rustup component add rustfmt
 
       - name: rustfmt
-        # TODO: change to `cargo fmt --check` after rustfmt 2.0 release
-        run: cargo fmt -- --check
+        run: cargo fmt --check
```

Comment:
I'm sure you know what's going on upstream far better than me given you actually work on rustfmt but it looks like the most recent version of rustfmt is 1.4.38. Did this feature get bumped forward?

Reply:
yup! it was something originally slated for a hypothetical 2.0 release, but was subsequently decided that a major/breaking version wouldn't make sense for tools shipped with the compiler/main distributable.

as such I decided to go ahead and pull this in